x/wasm/keeper/msg_dispatcher.go

```diff
@@ -188,12 +188,21 @@ func redactError(err error) error {
 	return fmt.Errorf("codespace: %s, code: %d", codespace, code)
 }
 
+func isIBCEvent(event sdk.Event) bool {
+	for _, attr := range event.Attributes {
+		if bytes.Contains(attr.Value, []byte("ibc_")) {
```

Comment:
Are we expecting "ibc_" at any place of string? I mean why not `HasPreffix`. According to the hermes, he is subscribed only on the `module` key. May be leave a link to the hermes's line with events description?